nat-lab/tests/test_mesh_firewall.py

```diff
@@ -713,33 +1051,53 @@ async def conntrack_on_stdout(stdout: str) -> None:
         time_wait_event = asyncio.Event()
 
         output_notifier.notify_output(
-            f"listening on [{CLIENT_ALPHA_IP}] {str(PORT)}", listening_start_event
+            f"Listening on {CLIENT_ALPHA_IP} {str(PORT)}", listening_start_event
```

Comment:
nit: maybe instead of fixing casing, it would be better to make the matching case insensitive?

Reply:
Made this an option in `OutputNotifier`.